src/main/java/technology/tabula/Page.java

```diff
@@ -48,6 +51,21 @@ public Page(float top, float left, float width, float height, int rotation, int
     }
 
     
+    public Page getArea(float top, float left, float bottom, float right, int mode) {
+        Rectangle area = new Rectangle(top, left, right - left, bottom - top);
+        return getArea(area, mode);
+    }
+
+    public Page getArea(Rectangle area, int mode) {
```

Comment:
I'd rather leave this method as it was, and make the _relative_→_absolute_ conversion in `CommandLineApp`.

Reply:
The orginal method is there as is... I didn't modify it. I simply added another method.
Additionally based on the comment at the top of the file...
~~~
// TODO: this class should probably be called "PageArea" or something like that
~~~
it seemed to me that Page.java is the place where all area calculations should be housed and hence I decided the add that method.

I can anyways move the calculations out of Page.java... but just want a quick confirmation from you again if possible please.

Reply:
Sorry about that. I misread and thought you modified the original method. My bad.

In any case, I still feel that the conversion belongs in `CommandLineApp`.